### PR TITLE
adds overdose to syndie medborg nanites

### PR DESCRIPTION
**Pull request by imsxz**
## About The Pull Request
few line changes idk what all the extra spacing is coming from tho

## Why It's Good For The Game
currently nukies can get medborgs to inject them with 500u nanites before they launch with no downsides for insane healing that wont run out, doesnt need the medborg to exist after that.

## Changelog
:cl: imsxz
balance: syndicate medborg nanites are now programmed to expel themselves from the body if they pass 30 units for the safety of the operatives.
/:cl:

### CODE REVIEW

**Comment by Tlaltecuhtli** [comment]
they spend tc for the shitty borg

**Comment by imsxz** [comment]
no french posters beyond this point

**Comment by Tlaltecuhtli** [comment]
ok?

**Comment by MrDoomBringer** [comment]
@imsxz the line spacing changes are cus your IDE automatically deletes extra tabs/whitespace. This is a good thing

**Comment by HomocidlePancakes** [comment]
MUH BOTANY THO

**Comment by HomocidlePancakes** [comment]
Also can you add OD to aranesp/increase toxin damage while you are at it?

**Comment by Kierany9** [comment]
I'd consider not making the overdose as insanely lethal or make it self-purging since you know some borg that's not too bright is still gonna spam click this and kill one of the operatives by accident (and then the rest of them in a chain reaction of implants going off) because if there's one thing I've learned is that not everybody reads the changelogs.

**Comment by imsxz** [comment]
fairly sure the medborgs beamgun is enough to outheal the damage from the overdose if it accidentally fucks up. for reference REM is 0.4 so the damage/healing isnt as large as it might seem

**Comment by ExcessiveUseOfCobblestone** [comment]
process 4 damage is pretty high.... did u test this?

**Comment by HomocidlePancakes** [comment]
So it heals 5 but deals 4. It will just outheal the OD

**Comment by imsxz** [comment]
yeah it ends up only doing 2 brute/burn per tick when you overdose, beamgun heals 4 each per tick so it can outheal the overdose if the medborg somehow manages to inject more than the hypospray holds before regenning.
tl;dr it's basically fucking impossible to accidentally overdose someone and if they do it can be dealt with easily

if you cant see the the balance issues with having an unending supply of very potent passive healing that /doesnt even need the medborg to be alive after you leave the shuttle/ i dunno what to tell you. It already gives you an advantage more than youd expect for 35TC not counting the fact that it has the beamgun and free kill on AI/comms, emag other borgs, shock doors, etc etc.

**Comment by ExcessiveUseOfCobblestone** [comment]
The hypo is from when the healing beam wasn't given to the mediborg (most likely before the gun even existed), if you're basically saying it's a second healing beam but without any of the balancing factors of said healing beam I don't know why you wouldn't just remove the chem.

Also I forget that OD still gives you the benefit of normal drugs rather than replacing the process (smh)

**Comment by imsxz** [comment]
if i tried to remove the chem we'd find out why i'm not trying to (IMPROVE DONT REMOVE) so instead i choose to improve it. your point about it serving the same purpose as the beamgun without any balancing factors is perfectly valid though, not sure what the whole idea was there.

**Comment by ExcessiveUseOfCobblestone** [comment]
Even with the nerf the issue would still remain, I don't have any issue with removing the hypo.

**Comment by imsxz** [comment]
the issues dont remain because you only get ~70 cycles (30 units at 0.4 metabolize rate) without borg injecting you compared to the normal ~500 cycles when the borg injects you a bunch while you're waiting to arrive on station. Also I don't want to get into an argument with anyone about the balance of medborgs active healing because right now WITH the hypo and beamgun it feels fine when the borg is following someone around and not dying themselves.

**Comment by GunHog** [comment]
If your intent is simply to prevent syndimedborgs from loading up the Syndies on the chem, why not just have the injector check if the nanites are present in the patient, stopping the injection if it does? Medibots have this feature already.

This method does not endanger the operatives if the borg player is incompetent or ignorant of this change.

**Comment by daklaj** [comment]
reduce the borg cost by 20 TC to compensate

**Comment by MrDoomBringer** [comment]
>reduce by 20tc
>15tc medborg
>warops spend 450tc on 30 fucking medborgs
>station gets medical malpracticed to death

**Comment by GuyonBroadway** [comment]
Better yet make the nanites overdose process cause them to rapidly self purge, no chemstacking, no need to fiddle with the hypospray, and also go away botany.

**Comment by Tlaltecuhtli** [comment]
wtf guyon is right

**Comment by ExcessiveUseOfCobblestone** [comment]
having a tlat upvote is a double edged sword

**Comment by GuyonBroadway** [comment]
I'll even throw in a lore reason. Syndie medical nanites contain a built-in self limiting system to prevent a grey goo scenario and/or to prevent their enemies from synthesizing it in any reasonable quantity.

**Comment by imsxz** [comment]
ok its now an annoying series of short stuns to vomit out all the nanites without hurting the operative, more annoying than harmful as it'll only take a few vomits to get it all out at 50u or however long it takes your medborg to realize the mistake

**Comment by imsxz** [comment]
this part wasnt tested btw :agony:

**Comment by Jimmius** [comment]
"it costs 35tc" isn't really a good enough excuse for unlimited powerful healing at all times with no way to stop it, for all your entire team, no matter what happens to the borg.

Any extra value you get from the medical borg is just a plus really, but even if it goes in and instantly dies to a 1 second timer flashbang, you've still got amazing value out of it from the nanites alone.
